File: windows/ManuscriptaTeacherApp/Main/Models/Enums/FeedbackStyle.cs
namespace Main.Models.Enums;

/// <summary>
/// Represents the feedback style for student device configuration.
/// See Validation Rules.md §2G(1)(b).
/// </summary>
public enum FeedbackStyle
{
    /// <summary>Correct/Incorrect feedback is shown immediately.</summary>
    IMMEDIATE,

    /// <summary>Only "Response Submitted" confirmation is shown.</summary>
    NEUTRAL
}

File: android/app/src/main/java/com/manuscripta/student/data/model/FeedbackStyle.java
package com.manuscripta.student.data.model;

/**
 * Enum representing the feedback style setting for quiz responses.
 * Per Validation Rules §2G(1)(b), determines how feedback is displayed to students.
 */
public enum FeedbackStyle {
    /**
     * Correct/Incorrect feedback is shown immediately after submitting.
     */
    IMMEDIATE,

    /**
     * Only "Response Submitted" confirmation is shown, without correctness feedback.
     */
    NEUTRAL
}
